src/Illuminate/Validation/Rules/Password.php

```diff
@@ -380,4 +380,37 @@ protected function fail($messages)
 
         return false;
     }
+
+    /**
+     * Get a regex representing the minimum password requirements.
+     *
+     * @return string
+     */
+    public function regex()
+    {
+        $regex = '^';
+
+        if ($this->letters) {
+            $regex .= '(?=.*[a-zA-Z])';
+        }
+
+        if ($this->mixedCase) {
+            $regex .= '(?=.*[a-z])(?=.*[A-Z])';
```

Comment:
This will fail on any non a-z characters.

It's very important that this method passes/fails on the same input as `passes()`.

`passes` uses https://www.php.net/manual/en/regexp.reference.unicode.php

If it's not possible to define a 100% equivalent regex for javascript then this `regex()` method might not be technically possible.